lib/src/config/config_file/mod.rs

```diff
@@ -252,7 +253,7 @@ pub struct MetricsServerSettings {
     pub bind: Option<String>,
     pub port: Option<u16>,
     pub username: Option<String>,
```

Comment:
If RPC username is sensitive, then metrics server username should, too?

Reply:
Done, but unsure if usernames should be hidden or not, see below.